### ISSUE TITLE
Training config for Real3D-AD

### ISSUE THREAD

**Issue by alibalapour** [yjnanan/PO3AD#5]
Hi,
Thanks for sharing your amazing work's code.

Based on train config you have provided, I assume this setting is not proper for Real3DAD dataset, as I cannot get the reported results. Can you please share your config code for Real3DAD dataset?

**Comment by alibalapour** [comment]
For example, I have run the same config you have provided for the Real3DAD Airplane category and got this result:
```
object AUC-ROC: 0.6656, point AUC-ROC: 0.6966393200116564, object AUCP-PR: 0.6892811062916988, point AUCP-PR: 0.0763858167318802
```

Can you please share the config you have used for the Real3DAD dataset?

**Comment by alibalapour** [comment]
Hi, 

I have a question regarding how you handled the inconsistency between the training and test sets in the Real3DAD dataset. As you know, the training dataset contains full scans of the 3D objects, while the test set consists of partial scans. Sometimes, the partial scans have coarse edges that may be considered anomalies or could pose challenges for the model in detecting anomalous regions. I would appreciate it if you could elaborate on this matter.
